src/node.cc

```diff
@@ -176,6 +176,8 @@ void WaitForInspectorDisconnect(Environment* env) {
 #if defined(__POSIX__) && !defined(NODE_SHARED_MODE)
     struct sigaction act;
     memset(&act, 0, sizeof(act));
+    sigemptyset(&act.sa_mask);
+    act.sa_flags = 0;
```

Comment:
Explicitly zeroing the `sa_flags` field isn't necessary with the memset() back.

Reply:
But indicates that we haven't forgotten about it / reminds reader about this field. Should I remove it?

Reply:
It's superfluous so yes please.